packages/docgen/src/test/fixtures/default-class-named/exports.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
How are these files generated?

Reply:
Using docgen's `--debug` [mode](https://github.com/WordPress/gutenberg/blob/master/packages/docgen/README.md#cli-options). 

From the root of the repo:

>npx docgen --debug packages/docgen/src/test/fixtures/default-class-named/code.js

Reply:
I seem to have tweaked the exports files removing the enclosing `[]`. In retrospect, don't know why I thought that was a good idea.